tools/Microsoft.Health.Tools.Sql.Tasks/Sql.test.targets

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <PropertyGroup>
+      <IntermediateOutputPath Condition=" '$(IntermediateOutputPath)' == '' ">$(MSBuildThisFileDirectory)obj\$(Configuration)</IntermediateOutputPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <AvailableItemName Include="TInitSqlScript" />
+        <AvailableItemName Include="TSqlScript" />
+        <AvailableItemName Include="SqlScript" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <GenerateFullScriptsInputs Include="@(TInitSqlScript)"/>
+        <GenerateFullScriptsInputs Include="@(TSqlScript)"/>
+        <GenerateFullScriptsInputs Include="@(SqlScript)"/>
+        <GenerateFullScriptsOutputs Include="$(GeneratedFullScriptPath)" />
+    </ItemGroup>
+
+   <Target Name="GenerateFullSqlScript" BeforeTargets="CoreCompile" Inputs="@(GenerateFullScriptsInputs)" Outputs="@(GenerateFullScriptsOutputs)">
+        <GenerateFullScript
+            IntermediateOutputPath="$(IntermediateOutputPath)"
+            OutputFile="$(GeneratedFullScriptPath)"
+            TInitSqlScript="@(TInitSqlScript)"
+            TSqlScript="@(TSqlScript)"
+            SqlScript="@(SqlScript)" />
+    </Target>
+
+    <UsingTask TaskName="GenerateFullScript" AssemblyFile="bin\$(Configuration)\publish\netstandard2.0\Microsoft.Health.Tools.Sql.Tasks.dll" />
```

Comment:
I'm not able to build because of this line - getting this error: 
![image](https://user-images.githubusercontent.com/88204686/136084190-900a6ec0-d247-4721-a7b7-368ba637075b.png)

Reply:
Fixed :-)